fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/FabricModel.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.datagen.v1;
+
+import net.minecraft.data.client.Model;
+
+import net.fabricmc.fabric.api.datagen.v1.builder.DisplayBuilder;
+import net.fabricmc.fabric.api.datagen.v1.builder.ElementBuilder;
+import net.fabricmc.fabric.api.datagen.v1.builder.OverrideBuilder;
+
+/**
+ * Fabric-provided extensions for {@link Model}.
+ *
+ * <p>Note: This interface is automatically implemented on all blocks via Mixin and interface injection.
```

Comment:
Probably incorrect :stuck_out_tongue: 
```suggestion
 * <p>Note: This interface is automatically implemented on all models via Mixin and interface injection.
```

Reply:
got caught lacking with javadoc 🥀